### PR TITLE
PLP add to cart functionality

### PR DESCRIPTION
**Pull request by NikhitaDesai**
### Problem 
Created PLP page as per the figma.
Monday Card Link :(add to cart functionality) https://marmeto-force.monday.com/boards/5104690360/pulses/5644380538
Preview link: https://mivi-website.myshopify.com/collections/earbuds?preview_theme_id=161946272021
### Solution
Added the relatable CSS properties to match the given figma
image
 ### Impacted Areas
<Explain other areas of the application that could be affected by this change.>
### Done Checklist

-  [x] Performed unit testing to ensure functionality and identify potential issues.
-  [x] Followed basic code standards and best practices. (indentation, media queries, basic customizer things)
-  [x] Added comments to explain complex logic or improve code readability.
-  [x] Followed Naming conventions and proper variable declaration.
-  [x] Handled edge cases and errors gracefully to prevent crashes.
-  [x] Optimized the code for production to ensure performance.
Please review this PR and provide feedback. Thank you!

### CODE REVIEW

File: assets/base.css
Comment (arshiya-khanam) [inline assets/base.css:1]: Remove commented code

File: snippets/mm-product-card.liquid
Comment (arshiya-khanam) [inline snippets/mm-product-card.liquid:1]: Indentation to be improved

**Review by arshiya-khanam — CHANGES_REQUESTED** [review]

File: assets/base.css
Comment (NikhitaDesai) [reply assets/base.css:1]: done

File: snippets/mm-product-card.liquid
Comment (NikhitaDesai) [reply snippets/mm-product-card.liquid:1]: done

**Review by arshiya-khanam — APPROVED** [review]

**Pull request merged**